src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/xunitrunner-perf.py

```diff
@@ -86,16 +118,19 @@ def _prepare_perf_execution_environment(settings, perf_runner):
         raise Exception("Failed to find perf runner {} in directory {}.".format(perf_runner, correlation_dir))
 
     # get the first subdir in the root and append it to xunit_perf_drop
-    buildSubdir = os.listdir(xunit_perf_drop)
-    xunit_perf_drop = os.path.join(xunit_perf_drop, buildSubdir[0])
-    xunit_perf_drop = os.path.join(xunit_perf_drop, "tools")
+    build_subdir = os.listdir(xunit_perf_drop)
+    xunit_perf_drop = os.path.join(xunit_perf_drop, build_subdir[0])
+    if perf_runner == 'Microsoft.DotNet.xunit.performance.runner.Windows':
+        xunit_perf_drop = os.path.join(xunit_perf_drop, "tools")
+    else:
+        xunit_perf_drop = os.path.join(*[xunit_perf_drop, "lib", "netstandard1.3"])
+
     log.info("Copying xunit perf drop from {} to {}.".format(xunit_perf_drop, test_location))
-    shutil.copy2(os.path.join(xunit_perf_drop, "xunit.performance.run.exe"), test_location)
-    shutil.copy2(os.path.join(xunit_perf_drop, "xunit.performance.metrics.dll"), test_location)
-    shutil.copy2(os.path.join(xunit_perf_drop, "xunit.performance.logger.exe"), test_location)
-    shutil.copy2(os.path.join(xunit_perf_drop, "xunit.runner.utility.desktop.dll"), test_location)
-    shutil.copy2(os.path.join(xunit_perf_drop, "ProcDomain.dll"), test_location)
-    shutil.copy2(os.path.join(xunit_perf_drop, "Microsoft.Diagnostics.Tracing.TraceEvent.dll"), test_location)
+
+    if osgroup.lower().startswith('windows'):
+        _prepare_windows_env_for_perf(xunit_perf_drop, test_location)
+    elif osgroup.lower().startswith('linux'):
+        _prepare_linux_env_for_perf(correlation_dir, xunit_perf_drop, test_location)
     # copy the architecture specific subdirectories
```

Comment:
And "else, log.error" is probably in order in case something unexpected is passed happens.